eng/WpfArcadeSdk/tools/Pbt.targets

```diff
@@ -92,12 +92,18 @@
           Condition="'$(InternalMarkupCompilation)'=='true' And Exists('$(LocalMicrosoftWinFXTargets)') "/>
 
   <!-- 
-    _WindowsDesktopSdkTargetFrameworkVersionFloor is defined in Microsoft.NET.WindowDesktop.props. 
-    This needs to be defined before Microsoft.NET.Sdk.WindowsDesktop.targets is imported. 
+    _WindowsDesktopSdkTargetFrameworkVersionFloor, _InvalidTargetFrameworkVersion & _TargetFrameworkVersionValue are defined in 
+     Microsoft.NET.WindowDesktop.props. 
+    
+     These need to be defined before Microsoft.NET.Sdk.WindowsDesktop.targets is imported. 
   -->
   <PropertyGroup Condition="'$(InternalMarkupCompilation)'=='true' And !Exists('$(LocalMicrosoftWinFXTargets)') ">
+    <_InvalidTargetFrameworkVersion>0.0</_InvalidTargetFrameworkVersion>
+    <_TargetFrameworkVersionValue Condition="'$(_TargetFrameworkVersionValue)' == ''">$([MSBuild]::ValueOrDefault($(_TargetFrameworkVersionWithoutV), $(_InvalidTargetFrameworkVersion)))</_TargetFrameworkVersionValue>
```

Comment:
Ignore the changes in Pbt.targets. 

I tried defining `_TargetFrameworkVersionValue` similar to here in `Microsoft.NET.Sdk.WindowsDesktop.props` (instead of defining an item `_WindowsDesktopTargetFrameworkVersion`), but it always ended up being `0.0`. 

Turns out that property evaluation order bites, and because SDK props are imported early in a project's life, `_TargetFrameworkVersionWithoutV` is never defined at the start when properties are being evaluated.

Reply:
Why should these be ignored? Your reasoning makes sense and defining this in targets seems reasonable.